components/aero_sensors/HW/HW_i2c.c

```diff
@@ -0,0 +1,260 @@
+/**
+  * @file HW_i2c.c
+  * @brief  Source code of I2C firmware/hardware interface
+  * @author Joshua Lafleur (josh.lafleur@outlook.com)
+  * @version 0.1
+  * @date 2022-07-09
+  */
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "SystemConfig.h"
+
+#include "HW_i2c.h"
+
+
+/******************************************************************************
+ *                           P U B L I C  V A R S
+ ******************************************************************************/
+
+I2C_HandleTypeDef i2c1;
+I2C_HandleTypeDef i2c2;
+DMA_HandleTypeDef hdma_i2c1_rx;
+DMA_HandleTypeDef hdma_i2c1_tx;
+DMA_HandleTypeDef hdma_i2c2_rx;
+DMA_HandleTypeDef hdma_i2c2_tx;
+
+
+/******************************************************************************
+ *                       P U B L I C  F U N C T I O N S
+ ******************************************************************************/
+
+
+/**
+ * @brief  Initializes the I2C1 bus
+ */
+void HW_I2C_Init(void)
+{
+    /**< Initialize peripherals */
+    i2c1.Instance            = I2C1;
+    i2c1.Init.ClockSpeed     = 100000U; /**< Clocked at 100kHz */
+    i2c1.Init.OwnAddress1    = 0x08;    /**< Translates to 0x10*/
```

Comment:
How does 0x10 translate to 0x08 here? Maybe just do the math instead of having a comment? Preprocessor logic here won't cost anything and will improve clarity